go.mod

```diff
@@ -1,49 +1,72 @@
 module github.com/dell/csi-baremetal-operator
 
-go 1.16
+go 1.17
 
 require (
 	github.com/Masterminds/semver v1.5.0 // indirect
-	github.com/dell/csi-baremetal v0.2.2-beta
-	github.com/go-logr/logr v0.3.0
+	github.com/dell/csi-baremetal v0.5.0-alpha.0.20220602080507-ad5d19149ddb
```

Comment:
Why did you choose v0.5.0?

Reply:
`go` takes last release tag, in this case it's `v0.5.0-alpha`.
This part `v0.5.0-alpha.0.20220602080507-ad5d19149ddb` called pseudo-version

Each and every pseudo version contains three parts. As in the above example go.mod file, we have v0.5.0-alpha.0.20220602080507-ad5d19149ddb for github.com/dell/csi-baremetal project where:

- `v0.5.0-alpha` is the Release Version of the package (v0.0.0 is the default release version when no tags are defined)
- `20220602080507 `is the Time of Commit of the package in UTC
- `ad5d19149ddb `are the first 12 characters from the commit hash of the package

Reply:
@rkiselenko thanks!
@mishoyama `v0.5.0-alpha` is the latest tag on master branch, `v1.0.0` is targeted to `release-1.0`